src/clj/rems/db/applications.clj

```diff
@@ -206,9 +223,10 @@
                               :when (= (:id app) (:appid e))]
                           ;; :appid needed only for batching
                           (dissoc e :appid))]
-         (assoc (get-application-state app app-events)
-                :formid (:formid (first catalogue-items))
-                :catalogue-items catalogue-items))))))
+         (enrich-application-description
```

Comment:
This is not good, because enrich-application-description makes way too many db calls.